definitions/npm/chalk_v2.x.x/flow_v0.25.x-/chalk_v2.x.x.js

```diff
@@ -1,105 +1,90 @@
-type $npm$chalk$StyleElement = {
-  open: string,
-  close: string
-};
+// From: https://github.com/chalk/chalk/blob/master/index.js.flow
 
-type $npm$chalk$Chain = $npm$chalk$Style & ((...text: any[]) => string);
+type TemplateStringsArray = $ReadOnlyArray<string>;
 
-type $npm$chalk$Style = {
-  // General
-  reset: $npm$chalk$Chain,
-  bold: $npm$chalk$Chain,
-  dim: $npm$chalk$Chain,
-  italic: $npm$chalk$Chain,
-  underline: $npm$chalk$Chain,
-  inverse: $npm$chalk$Chain,
-  strikethrough: $npm$chalk$Chain,
+export type Level = $Values<{
+  None: 0,
+  Basic: 1,
+  Ansi256: 2,
+  TrueColor: 3
+}>;
 
-  // Text colors
-  black: $npm$chalk$Chain,
-  red: $npm$chalk$Chain,
-  redBright: $npm$chalk$Chain,
-  green: $npm$chalk$Chain,
-  greenBright: $npm$chalk$Chain,
-  yellow: $npm$chalk$Chain,
-  yellowBright: $npm$chalk$Chain,
-  blue: $npm$chalk$Chain,
-  blueBright: $npm$chalk$Chain,
-  magenta: $npm$chalk$Chain,
-  magentaBright: $npm$chalk$Chain,
-  cyan: $npm$chalk$Chain,
-  cyanBright: $npm$chalk$Chain,
-  white: $npm$chalk$Chain,
-  whiteBright: $npm$chalk$Chain,
-  gray: $npm$chalk$Chain,
-  grey: $npm$chalk$Chain,
+export type ChalkOptions = {|
+  enabled?: boolean,
+  level?: Level
+|};
 
-  // Background colors
-  bgBlack: $npm$chalk$Chain,
-  bgBlackBright: $npm$chalk$Chain,
-  bgRed: $npm$chalk$Chain,
-  bgRedBright: $npm$chalk$Chain,
-  bgGreen: $npm$chalk$Chain,
-  bgGreenBright: $npm$chalk$Chain,
-  bgYellow: $npm$chalk$Chain,
-  bgYellowBright: $npm$chalk$Chain,
-  bgBlue: $npm$chalk$Chain,
-  bgBlueBright: $npm$chalk$Chain,
-  bgMagenta: $npm$chalk$Chain,
-  bgMagentaBright: $npm$chalk$Chain,
-  bgCyan: $npm$chalk$Chain,
-  bgCyanBright: $npm$chalk$Chain,
-  bgWhite: $npm$chalk$Chain,
-  bgWhiteBright: $npm$chalk$Chain
-};
+export type ColorSupport = boolean;
 
-declare module "chalk" {
-  declare var enabled: boolean;
-  declare var supportsColor: boolean;
+export interface Chalk {
```

Comment:
Is there a reason you're exporting it?

Reply:
No. I just copy the file from [chalk](https://github.com/chalk/chalk/blob/master/index.js.flow).
If you think this should not export, I will remove this.

Reply:
Ah, that's fine.